dm/dm.go

```diff
@@ -142,6 +161,10 @@ func execute() {
 
 	switch args[0] {
 	case "templates":
+		if len(args) < 2 {
+			fmt.Fprintln(os.Stderr, "No registry name supplied")
```

Comment:
I can do this change, but at that point it seems like we always pull it from the flag, or always pull it from the arguments. Pulling it from both places seems wrong to me.